napari/components/tests/test_viewer_model.py

```diff
@@ -81,7 +81,7 @@ def test_add_empty_points_to_empty_viewer():
 def test_add_empty_points_on_top_of_image():
     viewer = ViewerModel()
     image = np.random.random((8, 64, 64))
-    image_layer = viewer.add_image(image)
+    viewer.add_image(image)
```

Comment:
Yeah, these are annoying to me, mainly because `*_layer = viewer.add_*(data)` is a "canonical" form for napari so we would like to keep to it as much as possible. Perhaps we can test some aspect of the added layer, e.g. `assert image_layer.dims.ndim == 3`?

Reply:
yeah, i guess these are the annoyances of linting 😕
how do you feel about `_ = viewer.add_image(image)` here?  It's just a test after all (but I recognize that tests are a great place for people to learn to use it, and we'd like to establish conventions there)

Reply:
Yep, let's go for it! Indicating at least that something is returned is good enough for me!